### PR TITLE
Update to discrete summary checker

### PR DESCRIPTION
**Pull request by reedan88**

### CODE REVIEW

**Review by sauuyer — APPROVED** [review]
didn't run the notebook, but the code looks good.

File: discrete_summary_checker/example/Check_Format.ipynb [hunk 1]
```diff
@@ -1180,7 +451,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.9.12"
+   "version": "3.9.16"
   }
  },
  "nbformat": 4,
```
Comment (cooleyky) [inline discrete_summary_checker/example/Check_Format.ipynb:457]: Code looks good. I downloaded a summary spreadsheet from Alfresco and this notebook seems to work as intended.

File: discrete_summary_checker/example/Check_Format.py [hunk 2]
```diff
@@ -288,15 +296,14 @@ def is_same(series):
...
+    cruise_data = metadata[metadata["Cruise"] == cruise]
 
+    for station in cruise_data["Station"].unique():
+        # Get the data associated with a particular station
+        station_data = cruise_data[cruise_data["Station"] == station]
+    
+        # Run it through the validation checker
+        merrors = metadata_schema.validate(station_data)
+        for error in merrors:
+            print(error)
```
Comment (cooleyky) [inline discrete_summary_checker/example/Check_Format.py:309]: code looks good and works when tested in a terminal

**Review by cooleyky — APPROVED** [review]
Tried code locally and it all works as expected

**Pull request merged**